policies/design-process.md

```diff
@@ -41,6 +41,7 @@ Requirement for Design Documents
 
 A design document is required for any proposed enhancement which adds
 significant new APIs or non-trivially evolves or modifies existing APIs.
+A design document is also required all feature branches.
```

Comment:
"for all"?

Reply:
Ugh.  I read that sentence literally five or six times looking for mistakes 😢